### DISCUSSION TITLE
Mobile Navbar tab not aligned equally.

### DISCUSSION THREAD

**Discussion by adsingh14** [community/community#6185] [Mobile]
Bottom navbar buttons doesn't share same space.

Dpace between 3rd and 4th icon is more that the others.

![Screenshot_20211010-170128.png](https://user-images.githubusercontent.com/35252877/136693727-a8f5f2fc-2b6b-4121-98a6-9c7b8941a117.png)

**Comment by brianlovin** [comment]
Thanks for the note — yeah this is one of those funny things where they're mathematically distributed into 1/4th of the tab bar width, but because the label strings are longer or shorter, it messes up the optical distribution. I'm not sure if this is something we will prioritize fixing, but now I can't unsee 😅
